src/brod_group_subscriber_v2.erl

```diff
@@ -159,7 +159,7 @@
 %% brod_group_coordinator:start_link/6} Optional</li>
 %%
 %% <li>`consumer_config': For partition consumer,
-%% {@link brod_topic_subscriber:start_link/6}. Optional
+%% {@link brod_consumer:start_link/5}. Optional
```

Comment:
I'm sure that `consumer_config` option accepts config that is passed to `brod_consumer:start_link/5`, but here in `brod_group_consumer_v2` a link leads to `brod_topic_subscriber` where other irrelevant options are described.

Please, compare `brod_consumer` config:
https://hexdocs.pm/brod/brod_consumer.html#start_link/5
WIth `brod_topic_subscriber` config:
https://hexdocs.pm/brod/brod_topic_subscriber.html#start_link/1

Reply:
`consumer_group` is used here: https://github.com/kafka4beam/brod/blob/5172dbe5565bf1f234b8e5eaa7dc8924c1d3c05a/src/brod_group_subscriber_v2.erl#L510-L521

Reply:
What I am trying to say is that the link in the description of `consumer_config` option does not lead to a description of said options, which confused me a bit.

`consumer_config` options are eventually passed to [brod_consumer:start_link/5](https://hexdocs.pm/brod/brod_consumer.html#start_link/5) and I believe there should be a link to this function, because they are described in it's documentation.

But currently there is a link to [brod_topic_subscriber:start_link/1](https://hexdocs.pm/brod/brod_topic_subscriber.html#start_link/6) where docs describe only config of `brod_topic_subscriber`. [consumer_config](https://github.com/kafka4beam/brod/blob/5172dbe5565bf1f234b8e5eaa7dc8924c1d3c05a/src/brod_topic_subscriber.erl#L236-L238) is one of the options in `brod_topic_subscriber`'s config, but even there a link leads to [brod_topic_subscriber](https://hexdocs.pm/brod/brod_topic_subscriber.html#start_link/6) itself. I'm going to fix this too in this PR.

In addition, in other places like in [brod_group_subscriber](https://github.com/kafka4beam/brod/blob/5172dbe5565bf1f234b8e5eaa7dc8924c1d3c05a/src/brod_group_subscriber.erl#L199-L200) description of `ConsumerConfig` option has a link to `brod_consumer` docs as I'm suggesting.